src/services/ApiService.js

```diff
@@ -0,0 +1,31 @@
+import { Config } from "@baltimorecounty/javascript-utilities";
+import axios from "axios";
+
+const { getValue } = Config;
+
+/**
+ * Determines if api is up or not
+ * @returns true if the api is available
+ */
+const GetStatus = () =>
+  axios
+    .get(`${getValue("apiRoot")}/status`)
+    .then(({ status }) => status === 200);
+
+/**
+ * Get Services Data from SiteExecutive structured content
+ */
+const GetServices = () =>
+  axios
+    .get(`${getValue("apiRoot")}/services`)
+    .then(({ status, data }) => (status === 200 ? data : []));
+
+/**
+ * Get Services Data from SiteExecutive structured content
+ */
+const ClearServicesCache = () =>
```

Comment:
this can be removed, the clearing of cache is a utility for webservices when they add / modify a service, and will be trigged manually.

Reply:
done